packages/url_launcher/url_launcher_android/test/url_launcher_android_test.dart

```diff
@@ -415,7 +431,15 @@ class _FakeUrlLauncherApi implements UrlLauncherApi {
   @override
   Future<bool> launchUrl(String url, Map<String?, String?> headers) async {
     passedWebViewOptions = WebViewOptions(
-        enableJavaScript: false, enableDomStorage: false, headers: headers);
+      enableJavaScript: false,
+      enableDomStorage: false,
+      headers: headers,
+    );
+
+    passedBrowserOptions = BrowserOptions(
```

Comment:
Why is this being set?

Reply:
You are right, this doesn't make sense. Removed that